tests/st/calicoctl/test_crud.py

```diff
@@ -1479,369 +1479,366 @@ def test_get(self):
+                                            }],
+                                'Ingress': [{'action': 'Allow',
+                                             'destination': {},
+                                             'protocol': 'UDP',
+                                             'source': {}}],
+                                'order': 100000,
+                                'selector': ""}}),
+                   ("policy-invalidLowPortinRange", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'NetworkPolicy',
+                       'metadata': {'name': 'policy2'},
+                       'spec': {'Egress': [{'action': 'Deny',
+                                            'destination': {},
+                                            'source': {
+                                                'ports': [0-65535],  # Min port is 1
+                                                'protocol': 'TCP',
+                                            },
+                                            }],
+                                'Ingress': [{'action': 'Allow',
+                                             'destination': {},
+                                             'protocol': 'UDP',
+                                             'source': {}}],
+                                'order': 100000,
+                                'selector': ""}}),
+                   ("policy-invalidLowPortinList", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'NetworkPolicy',
+                       'metadata': {'name': 'policy2'},
+                       'spec': {'Egress': [{'action': 'Deny',
+                                            'destination': {},
+                                            'source': {
+                                                'protocol': 'TCP',
+                                                'ports': [0, 10, 80]  # Min port is 1
+                                            },
+                                            }],
+                                'Ingress': [{'action': 'Allow',
+                                             'destination': {},
+                                             'protocol': 'UDP',
+                                             'source': {}}],
+                                'order': 100000,
+                                'selector': ""}}),
+                   ("policy-invalidReversedRange", {
+                       'apiVersion': API_VERSION,
+                       'kind': 'NetworkPolicy',
+                       'metadata': {'name': 'policy2'},
+                       'spec': {'Egress': [{'action': 'Deny',
+                                            'destination': {},
+                                            'source': {
+                                                'protocol': 'TCP',
+                                                'ports': [65535-1]  # range should be low-high
```

Comment:
I don't think that is valid format for port range, range is separated by a colon.

And i think it would need to be in quotes too.